src/components/TxModal.vue

```diff
@@ -308,6 +314,10 @@ export default {
           break;
         case 'insufficient':
         case 'failed':
+          logTrackerEvent(this, 'NFT', 'ClickRetry', this.classId, 1);
```

Comment:
I suggest to use more specific event name and use snake_case, e.g. `tx_modal_retry_button_clicked`